sdks/sdk-core/src/addresses.ts

```diff
@@ -303,8 +303,8 @@ const UNICHAIN_ADDRESSES: ChainAddresses = {
   tickLensAddress: '0xd5d76fa166ab8d8ad4c9f61aaa81457b66cbe443',
   swapRouter02Address: '0x73855d06de49d0fe4a9c42636ba96c62da12ff9c',
 }
-
-export const CHAIN_TO_ADDRESSES_MAP: Record<SupportedChainsType, ChainAddresses> = {
+// exclude monad because it doesn't have v3 support
+export const CHAIN_TO_ADDRESSES_MAP: Omit<Record<SupportedChainsType, ChainAddresses>, ChainId.MONAD_TESTNET> = {
```

Comment:
change to `Omit<Record<SupportedChainsType, ChainAddresses>, ChainId.MONAD_TESTNET> ` doesn't seem like a great solution. So far we have relied on SupportedChainsType enum strong type to ensure we don't miss an address on any supported chain.

Reply:
Removed this to unblock urgent wallet PR but we'll need to revisit (maybe launch v3 anyways?) for web